src/Pact/Native/Keysets.hs

```diff
@@ -49,6 +53,14 @@ keyDefs =
      "Define keyset as NAME with KEYSET, or if unspecified, read NAME from message payload as keyset, \
      \similarly to 'read-keyset'. \
      \If keyset NAME already exists, keyset will be enforced before updating to new value."
+    ,setTopLevelOnly $ defGasRNative "describe-keyset" descKeySet
+     (funType tTyObjectAny [("keyset",tTyString)])
+     ["(env-data {\"k\": { \"pred\": \"keys-all\", \"keys\": [\"abc\"] }) (describe-keyset (read-keyset 'k))"]
```

Comment:
Not sure why this example is failing the test suite.

Reply:
Here's working code (will need quote escapes):
```
(env-data {"k": ["abc"] }) (define-keyset 'k (read-keyset 'k)) (describe-keyset 'k)
```